setup.py

```diff
@@ -110,17 +115,20 @@ def run(self) -> None:
                     zip.writestr("playwright/driver/README.md", "Universal Mac package")
 
         os.remove(base_wheel_location)
-        for whlfile in glob.glob(os.path.join(self.dist_dir, "*.whl")):
-            os.makedirs("wheelhouse", exist_ok=True)
-            with InWheel(
-                in_wheel=whlfile,
-                out_wheel=os.path.join("wheelhouse", os.path.basename(whlfile)),
-                ret_self=True,
-            ):
-                print("Updating RECORD file of %s" % whlfile)
-        shutil.rmtree(self.dist_dir)
-        print("Copying new wheels")
-        shutil.move("wheelhouse", self.dist_dir)
+        if InWheel is not None:
```

Comment:
```suggestion
        if InWheel:
```
not sure if its cleaner.